autoload/neomake/makers/ft/go.vim

```diff
@@ -1,16 +1,22 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers()
-    return ['go', 'golint', 'govet']
+    return ['gotest', 'golint', 'govet']
```

Comment:
I'd say to not change the name, but keep it as "go".
Or at least still provide the old maker function for backward compatibility.